R/rapporttekst-funksjoner.R

```diff
@@ -170,9 +170,40 @@ akse_tall_format = function(antall_desimaler = 2, decimal.mark = ",", big.mark =
 #' @description
 #' `r lifecycle::badge("experimental")`
 #'
-#' Tar inn et tall og konverterer det til LaTeX-kommando for å skrive prosent-tegn i tekst.
-#' @param x Tallet som skal skrives som prosentverdi.
-#' @param desimalar Antall desimaler som skal vises.
+#' Tar inn en vektor med tall som konverteres til prosent
+#' og gir ut en vektor med LaTeX-kommandoer for finformatert
+#' visning av tallene med prosenttegn.
+#' Tallene blir avrundet til et bestemt antall desimaler.
+#' 
+#' @param x Vektor med tallene en ønsker å konvertere til prosent og få ut på LaTeX-format.
+#' @param desimalar Antall desimaler som skal vises etter kommaet.
+#' 
+#' @details
+#' Tallene blir konvertert til prosent og blir vist med prosenttegn
+#' («0.1234» blir «12.34 %»);
+#' desimaltegnet blir (dersom språket er norsk)
+#' komma («3,14»),
+#' istedenfor punktum («3.14»);
+#' negative tall blir vist med ekte minustegn («−42»),
+#' istedenfor bindestrek («-42»);
+#' og `NA`-verdier blir vist som en kort tankestrek («–»).
```

Comment:
Dette stemmer ikkje. Poenget er at ein får ut *LaTeX-kode* for formatering av talet som prosent dersom ein brukar `kvalreg`-klassen vår. Det *kan* stå forklart kva resultatet av LaTeX-koden, men det blir berre ekstrainformasjon.

Reply:
Forresten, det treng ikkje stå forklart her i det heile tatt; ein kan berre visa til hjelpefila for `num()`-funksjonen for detaljar.